ddl/db_test.go

```diff
@@ -4267,6 +4267,34 @@ func (s *testDBSuite2) TestTransactionOnAddDropColumn(c *C) {
 	tk.MustQuery("select a,b from t1 order by a").Check(testkit.Rows("1 1", "1 1", "1 1", "2 2", "2 2", "2 2"))
 }
 
+func (s *testDBSuite3) TestIssue22307(c *C) {
```

Comment:
I think that we may meet a case where cases trying to modify the hook simultaneously, may we move to the serial test?

Reply:
@xiongjiwei  ping on the PR.

Reply:
seems that every `testDBSuite`  has its own `domain`, so the hook may not be affected by other tests

Reply:
Tests in one suite may be parallel.